assets/src/edit-story/components/library/panes/media/karma/mediaFetching.karma.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+
+import { Fixture, MEDIA_PER_PAGE } from '../../../../../karma/fixture';
+import { ROOT_MARGIN } from '../mediaPane';
+
+describe('MediaPane fetching', () => {
+  let fixture;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    await fixture.render();
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  it('should fetch 2nd page', async () => {
+    const mediaLibrary = fixture.querySelector('[data-testid="mediaLibrary"]');
+    let mediaElements = fixture.querySelectorAll('[data-testid=mediaElement]');
+
+    expect(mediaElements.length).toBe(MEDIA_PER_PAGE);
+
+    mediaLibrary.scrollTo(
+      0,
+      mediaLibrary.scrollHeight - mediaLibrary.clientHeight - ROOT_MARGIN
+    );
+    await new Promise((r) => setTimeout(r, 10));
```

Comment:
Should this one just work? changing default interval/timeout doesn't help. 
https://github.com/google/web-stories-wp/pull/2314/commits/bc0d1cd346a5ebd824b6be1046e573eea862a029 @dvoytenko

Reply:
Not sure what's the problem there is. If we use a native DOM event or setTimeout, they can be wrapped in the `useBatchingCallback` callback. But it's also ok to do `waitFor` in some rare cases.

Reply:
I think `useBatchingCallback` will not help us here (on the editor side). `waitFor` should work here (because it's just setInterval+setTimeout) but there is something wrong with it or timers in general, when I put `{ timeout: 10000 }` (or any combination of interval/timeout) it will not wait, but it should timeout on global 5000ms timer.
Any idea why interval/timeout option doesn't work?